qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -53,7 +50,7 @@ class AdmissionControllerTest extends BaseSpecification {
     static final private Deployment NGINX_DEPLOYMENT = new Deployment()
             .setName(NGINX)
             .setNamespace(TEST_NAMESPACE)
-            .setImage(NGINX_IMAGE)
+            .setImage(TEST_IMAGE)
```

Comment:
This was setting an image without sha, and now we're passing one **with** a sha. Similar elsewhere.
Maybe it matters in some of these places, maybe not...

Reply:
Correct, I switched it to sha to not download it again. That's the purpose of this change, limit interactions with quay.io

Reply:
Reverted changes to this file